packages/contentstack-import-setup/src/utils/common-helper.ts

```diff
@@ -0,0 +1,21 @@
+export const formatError = (error: any) => {
+  try {
+    if (typeof error === 'string') {
+      error = JSON.parse(error);
+    } else {
+      error = JSON.parse(error.message);
+    }
+  } catch (e) {}
+  let message = error?.errorMessage || error?.error_message || error?.message || error;
+  if (error && error.errors && Object.keys(error.errors).length > 0) {
+    Object.keys(error.errors).forEach((e) => {
+      let entity = e;
+      if (e === 'authorization') entity = 'Management Token';
+      if (e === 'api_key') entity = 'Stack API key';
+      if (e === 'uid') entity = 'Content Type';
+      if (e === 'access_token') entity = 'Delivery Token';
+      message += ' ' + [entity, error.errors[e]].join(' ');
+    });
+  }
+  return message;
+};
```

Comment:
@shafeeqd959 , I think we can remove this file and directly use formatError functionality from utilities plugin.
https://github.com/contentstack/cli/blob/main/packages/contentstack-utilities/src/helpers.ts#L67

Reply:
@aman19K this will take up in the next release as we are in the last minute